ayon_api/entity_hub.py

```diff
@@ -3332,11 +3432,11 @@ def set_assignees(self, assignees):
 
     assignees = property(get_assignees, set_assignees)
 
-    def add_child(self, child):
+    def add_child(self, child: BaseEntity) -> None:
```

Comment:
This is overriding super where it is `Union[BaseEntity, str]`
```suggestion
    def add_child(self, child: Union[BaseEntity, str]) -> None:
```